db/data/20220930022007_populate_room_limit_role_permission.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class PopulateRoomLimitRolePermission < ActiveRecord::Migration[7.0]
+  def up
+    Permission.create! [
+      { name: 'RoomLimit' }
+    ]
+
+    admin = Role.find_by(name: 'Administrator')
+    user = Role.find_by(name: 'User')
+    guest = Role.find_by(name: 'Guest')
+
+    room_limit = Permission.find_by(name: 'RoomLimit')
+
+    RolePermission.create! [
+
+      { role: admin, permission: room_limit, value: 100 },
```

Comment:
I think it's better to make the values strings to stay consistent, what do you think?

Reply:
They will be casted into a string when saving the record but I believe it's better to be consistent with the rest of role permissions creation.

Reply:
WIll do in next PR